concent_api/core/migrations/0010_auto_20180918_1036.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-09-18 10:36
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0009_auto_20180829_1224'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='pendingresponse',
+            name='response_type',
+            field=models.CharField(choices=[('ForceReportComputedTask', 'ForceReportComputedTask'), ('ForceReportComputedTaskResponse', 'ForceReportComputedTaskResponse'), ('VerdictReportComputedTask', 'VerdictReportComputedTask'), ('ForceGetTaskResultFailed', 'ForceGetTaskResultFailed'), ('ForceGetTaskResultUpload', 'ForceGetTaskResultUpload'), ('ForceGetTaskResultDownload', 'ForceGetTaskResultDownload'), ('ForceSubtaskResults', 'ForceSubtaskResults'), ('SubtaskResultsSettled', 'SubtaskResultsSettled'), ('ForceSubtaskResultsResponse', 'ForceSubtaskResultsResponse'), ('SubtaskResultsRejected', 'SubtaskResultsRejected'), ('ForcePaymentCommitted', 'ForcePaymentCommitted')], max_length=32),
+        ),
+    ]
```

Comment:
No. There shouldn't be any. If there are we should get a big and loud crash and investigate because it would mean that there's a problem we do not understand. As far as we know messages like this were always being returned synchronously and never even reached `receive/`